docs/apis-clients/web-modeler-api/index.md

```diff
@@ -0,0 +1,76 @@
+---
+id: index
+title: Web Modeler API (REST, beta)
+description: "Web Modeler API (beta) is a REST API and provides access to Web Modeler data.
+Requests and responses are in JSON notation."
+---
+
+:::caution Beta Offering
+Web Modeler API is currently offered as a [beta release](../../reference/early-access#beta). It is not recommended for production use and there is no maintenance service guaranteed.
+
+While in beta, the API may introduce breaking changes without prior notice.
+
+However, we encourage you to provide feedback via your designated support channel or the [Camunda Forum](https://forum.camunda.io/).
+:::
+
+Web Modeler provides a REST API at `/api/*`. Clients can access this API by passing a JWT access token in an authorization header `Authorization: Bearer <JWT>`.
+
+## OpenAPI documentation
+
+A detailed API description is available as [OpenAPI](https://www.openapis.org/) specification at [https://modeler.cloud.camunda.io/swagger-ui/index.html](https://modeler.cloud.camunda.io/swagger-ui/index.html).
+
+## Authentication
+
+To authenticate for the API, generate a JWT token and pass it in each request; guidance on this is provided in the following sections.
+
+### Obtain JWT token
+
+**Example:**
+
+1. Create client credentials by clicking **Console > Manage (Organization) > Console API > Create New Credentials**.
+2. Add permissions to this client for **Web Modeler API (beta)**.
+3. After creating the client, you can download a shell script to obtain a token.
```

Comment:
How do you download this?

Reply:
You click on the download button that appears after you clicked "Create" in the last UI. I admit that it is not clear if you only read this text, but if you read this text and do it at the same time, it should be pretty obvious. Also same comment applies to missing documentation on creating Console API credentials as above:

![image](https://user-images.githubusercontent.com/14032870/209139863-811fa40c-04da-499b-b828-1b96be2f7692.png)